validator/sawtooth_validator/concurrent/atomic.py

```diff
@@ -128,3 +128,32 @@ def get(self, key, default):
                 return self._dict[key].copy()
             except KeyError:
                 return default
+
+
+class ConsensusRegistry:
```

Comment:
This class should go in a new file in the `consensus/` folder. `concurrent/` is just for concurrency primitives missing from the python standard library.